### DISCUSSION TITLE
JPA에서 orphanRemoval을 사용할 시 발생하는 문제

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#152] [1. 개발 관련 글 모음]
https://velog.io/@shkisme/JPA-cascadeType.REMOVE%EC%99%80-orphanRemoval-%EC%82%AC%EC%9A%A9-%EC%8B%9C-%EC%84%B1%EB%8A%A5-%EB%AC%B8%EC%A0%9C

다음에는 어떤 글을 써볼까요 🤔

**Comment by amaran-th** [comment]
오오 잘 읽었습니다!
이런 이슈가 있었군요...
글을 이해하기 쉽게 잘 쓰시는 것 같아요😆

**Reply by shkisme** [reply]
감사합니당 😚
